packages/angular/src/toolkit/table-settings/table-settings.stories.ts

```diff
@@ -0,0 +1,158 @@
+import { Component, Input } from '@angular/core';
+import { withKnobs } from '@storybook/addon-knobs';
+import { moduleMetadata, storiesOf } from '@storybook/angular';
+import { SortableList, SortableListOption } from '../sortable-list';
+import { CheckboxSetting, ComponentSetting, RadioSetting } from './settings';
+import { TableSettings } from './table-settings-model.class';
+import { TableSettingsPane } from './table-settings-pane.class';
+import { TableSettingsModule } from './table-settings.module';
+
+@Component({
+  selector: 'sc-story-custom-component',
```

Comment:
I don't see how this would be a useful change in this case ... I kinda like the firewalling we've given the `toolkit/` directory with the `sc` namespace - keeps all that weirdness isolated